packages/ui/src/components/UserBox.svelte

```diff
@@ -58,43 +53,45 @@
         event.stopPropagation()
       }}
     >
-      {#if selected}
-        <div class="avatar"><UserInfo user={selected.name} size={36} avatarOnly /></div>
+      {#if selectedItem}
+        <div class="avatar"><UserInfo user={selectedItem} size={36} avatarOnly /></div>
       {:else}
         <div class="icon">
           {#if pressed}<Close />{:else}<Add />{/if}
         </div>
       {/if}
     </button>
     <div slot="header" class="search"><EditBox label={ui.string.Search} bind:value={search} /></div>
-    {#if selected}
+    {#if selectedItem}
       <PopupItem
         component={UserInfo}
-        props={{ user: selected.name }}
+        props={{ user: selectedItem }}
         selectable
         selected
         action={async () => {
           selected = undefined
           pressed = !pressed
+          dispatch('change')
         }}
       />
     {/if}
-    {#each users.filter((u) => u !== selected && u.title.toLowerCase().indexOf(search.toLowerCase()) !== -1) as user}
+    {#each users.filter((u) => u._id !== selected && u.name.toLowerCase().indexOf(search.toLowerCase()) !== -1) as user}
       <PopupItem
         component={UserInfo}
-        props={{ user: user.name }}
+        props={{ user: user }}
         selectable
         action={async () => {
-          selected = user
+          selected = user._id
           pressed = !pressed
+          dispatch('change')
         }}
       />
     {/each}
   </PopupMenu>
   <div class="selectUser">
     <div class="title"><Label label={title} /></div>
     <div class="user">
-      {#if selected}{selected.title}{:else}<Label {label} />{/if}
+      {#if selectedItem}{selectedItem.name}{:else}<Label {label} />{/if}
```

Comment:
You should also show _id I think.

Reply:
I think that `_id` may be user e-mail or something else, and `name` it be fullName or nickName, or anything that user whant to be showing

Reply:
Yes _id will be email or accountId.